modules/loader-utils/src/lib/request-utils/request-scheduler.js

```diff
@@ -30,7 +30,7 @@ export default class RequestScheduler {
     // Tracks the number of active requests and prioritizes/cancels queued requests.
     this.requestQueue = [];
     this.activeRequestCount = 0;
-    this.requestMap = {};
+    this.requestMap = new Map();
```

Comment:
General preference for maps?

Reply:
I think the use of `Map` is because there's no longer a separate identifier.

> `handle` is no longer required to have an `id`

Instead the `handle` itself _is the key_ of the `Map`